### ISSUE TITLE
Deno cannot compile to a file that has no `.ts` in the import

### ISSUE THREAD

**Issue by thoriqadillah** [denoland/deno#26102]
Version: Deno 2.0.0
I tried to play around with deno 2.0 by compiling my nest js project with deno. I thought maybe if i can compile the project into one executable binary that would be cool. Its pretty complex project. But i found that deno cannot compile if my import has no `.ts` on it. Maybe its obvious, but that would take a tedious task to add `.ts` in almost every imports

this is the error message
```bash
deno compile -R src/main.ts --unstable-sloppy-imports
error: Module not found "file:///project-dir/src/sentry". Maybe add a '.ts' extension or run with --unstable-sloppy-imports
    at file:///project-dir/src/main.ts:1:8
```

**Comment by bartlomieju** [comment]
You need to put `--unstable-sloppy-imports` after the `-R` flag. Currently this flag is forwarded to `src/main.ts` as a value of `Deno.args`.

**Comment by dsherret** [comment]
Yeah, as the warning states, sloppy imports aren't yet supported in `deno compile`.

> that would take a tedious task to add .ts in almost every imports

`deno lint --unstable-sloppy-imports --fix` automatically adds the extensions
